src/redisearch_rs/inverted_index/src/lib.rs

```diff
@@ -393,18 +377,54 @@ impl<'a> IntoIterator for RSAggregateResult<'a> {
 #[derive(Debug, PartialEq)]
 pub struct RSVirtualResult;
 
+#[bitflags]
+#[repr(u32)]
```

Comment:
Could we use  `#[repr(u8)]` instead?
I suppose we'd have to change `RSResultTypeMask` as well.